lib/gaffe/errors.rb

```diff
@@ -3,8 +3,13 @@ module Errors
     extend ActiveSupport::Concern
 
     included do
-      before_filter :fetch_exception, only: %w(show)
-      before_filter :append_view_paths
+      if Rails::VERSION::MAJOR >= 5
+        before_action :fetch_exception, only: %w(show)
+        before_action :append_view_paths
+      else
+        before_filter :fetch_exception, only: %w(show)
+        before_filter :append_view_paths
+      end
```

Comment:
Oh, I think this might be a good time to drop support for Rails 3 (and 1.9.3) then. Rails 3 is not supported by the Rails team anymore.